tests/snuba/test_deletes.py

```diff
@@ -0,0 +1,77 @@
+import time
+from datetime import datetime, timedelta
+from uuid import uuid4
+
+import pytest
+from snuba_sdk import DeleteQuery, Request
+
+from sentry.snuba.dataset import Dataset
+from sentry.testutils.cases import SnubaTestCase, TestCase
+from sentry.utils import snuba
+
+
+class SnubaTest(TestCase, SnubaTestCase):
+    @pytest.mark.xfail
+    def test_basic(self) -> None:
+        # insert a new issue
+        now = datetime.now()
+        id = uuid4()
+        issue = (
+            2,
+            "insert",
+            {
+                "group_id": 1,
+                "message": "message",
+                "platform": "python",
+                "primary_hash": "1" * 32,
+                "event_id": "a" * 32,
+                "project_id": self.project.id,
+                "datetime": now.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
+                "data": {"received": time.mktime(now.timetuple())},
+                "occurrence_data": {
+                    "detection_time": time.mktime(now.timetuple()),
+                    "fingerprint": ["hi"],
+                    "issue_title": "myissue",
+                    "id": id,
+                    "type": 1,
+                },
+                "organization_id": 6,
+            },
+        )
+        self.store_issues([issue])
+
+        # make sure its there
+        assert snuba.query(
+            dataset=Dataset.IssuePlatform,
+            start=now - timedelta(days=1),
+            end=now + timedelta(days=1),
+            groupby=["project_id"],
+            filter_keys={"project_id": [self.project.id]},
+            referrer="testing.test",
+            tenant_ids={"referrer": "testing.test", "organization_id": 1},
+        ) == {self.project.id: 1}
+
+        # delete it
+        req = Request(
+            dataset="search_issues",
+            app_id="my_app",
+            query=DeleteQuery(
+                "search_issues", {"project_id": [self.project.id], "occurrence_id": [str(id)]}
+            ),
+        )
```

Comment:
any reason you aren't adding the tenant_ids as well? I guess it's super important for the tests but since this test helps document what the delete query should look like might as well add them

Reply:
I added tenant ids to the test.

that makes sense, id assume it not being enforced yet is also the reason its not failing yet. but im not sure where the allocation policy enforcement actually happens in the code.